Python/pdf_reorder/pdf_reorder_.py

```diff
@@ -0,0 +1,50 @@
+
+from pdfrw import PdfReader, PdfWriter
+
+# The function input_and_parse gathers the inputs and parses and then sorts them in
+# the order required by reorder(path,dic) function
+#It takes one argument which is page length of pdf 'n'
+ 
+def input_and_parse(n):
+    print("="*20)
+    print("enter the current page and the page you want it to be on seperate values by a comma ',' \n") 
+    # store the input in a list and then convert the input string into 
+    #using map function to convert the data in lists into int values

```

Comment:
Keep the Comments and spacing consistent.

Reply:
used autopep and it is done.